fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadJob.java

```diff
@@ -760,18 +760,27 @@ public List<Comparable> getShowInfo() throws DdlException {
             jobInfo.add(state.name());
 
             // progress
+            // check null
+            ProgressManager.Progress progressShow = Env.getCurrentProgressManager().getProgressClass(id);
+            String progress = "ERROR";
+            if (progressShow != null) {
```

Comment:
Please extract a method for this logic and move it to `ProgressManager.Progress`